src/MembershipsSDKAdapter.js

```diff
@@ -119,11 +146,17 @@ export default class MembershipsSDKAdapter extends MembershipsAdapter {
       host: null,
     });
 
-    const members$ = defer(() => this.datasource.memberships.list({
+    const me$ = from(this.datasource.people.get('me'));
+
+    const memberships$ = defer(() => this.datasource.memberships.list({
       roomId: roomID,
       max: MAX_MEMBERSHIPS,
     })).pipe(
-      map((page) => page.items.map(membershipToMember)),
+      map((page) => page.items),
+    );
+
+    const orderedMembers$ = combineLatest([me$, memberships$]).pipe(
```

Comment:
you can now rename `orderedMembers$` to `members$` , to revert some of the changes below.